src/containers/Home/Home.css

```diff
@@ -0,0 +1,8 @@
+.Home-members {
```

Comment:
we should not use class-names starting with capital-letters.

Reply:
@uday-kybing  recommended this